x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -21,7 +21,7 @@ export function registerIndexRoutes({ router }: RouteDependencies) {
     async (context, _, response) => {
       const { client } = (await context.core).elasticsearch;
       try {
-        const indices = await fetchIndices(client, 'search-*', false, /^search-.*/);
+        const indices = await fetchIndices(client, '*', false);
```

Comment:
question: Awesome that we can get rid of the RegEx! Do you think it would be possible to unify this route and the route below, or are there too many significant functional differences?

Reply:
That's a great question. We might be able to. I'm happy to look into it... is it something we can do in a follow-up PR, or would you rather us change it here?

Reply:
A follow-up PR would work fine, I think.